lib/uktt/parser/json_api.rb

```diff
@@ -29,8 +29,8 @@ def parse_resource(resource)
         result = {}
 
         parse_top_level_attributes!(resource, result)
-
         parse_relationships!(resource['relationships'], result) if resource.key?('relationships')
+        parse_meta!(resource, result)
```

Comment:
Following the same primadonna idiom for now. I think we'll probably refactor this class at some stage.

Reply:
Do we have a unit test that could indirectly test this?